anti.md

```diff
@@ -11,34 +11,78 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Some runners (and similarly cyclists, hikers, and swimmers) like to save routes so they don't have to memorize
+each path (i.e., where to turn) and corresponding distance.
+The ability to reference saved routes saves effort and brain power
+and is particularly convenient when choosing a route based on distance.
+Today, apps/sites like [Runkeeper] allow users to trace a route on a map,
+and then the app/site saves the route and calculates the distance.
+This method is a good solution to the problem, and I think an argument can be made that this route creation system is itself a DSL,
+but I think a more traditional DSL with more conventional syntax would not be/have been an appropriate solution to the problem.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+I think a more traditional, 'human-readable' DSL would _not_ be appropriate because,
+assuming the user is interfacing with a file rather than a map,
+there would need to be some way to represent 'actual space' like streets;
+but I'm not convinced that conventional syntax offers a natural and simple way to represent relational points in space.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I hit a 'running peak' last semester, and this semester I'd like to revisit and surpass that peak.
+I'm not quite starting from scratch, but (especially since it's Summer still) I'm not ready to go out
+and run as far as I was willing and able to last semester;
+so I'm building up my mileage gradually. I'll often decide what distance I want to run before I start,
+then browse my repository of routes, and choose one that matches the intended distance.
+I was content with my experience in creating the routes,
+but the perfectionist in me wished that the routes could be more precise
+than a rough trace&mdash;hence I (briefly) considered a language.
+
```

Comment:
!! That was my other thought about this. It would be nice to opt in to turn-by-turn directions for routes for which you're unfamiliar. If you want the directions, it could maybe try to compute them based on the street.